app/code/community/Yotpo/Yotpo/Helper/Data.php

```diff
@@ -53,14 +53,7 @@ private function renderYotpoProductBlock($thisObj, $blockName, $product = null,
 
         if ($product != null) {
             $block->setAttribute('product', $product);
-        }
```

Comment:
@ijazalise my mistake, please return the logic as it was. sorry for confusing you

Reply:
ok